tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -242,51 +269,63 @@ def test_postgres(
         # create table
         table_factory(engine=postgres_ds.get_engine(), table_names={table_name})
 
-        table_names: list[str] = inspect(postgres_ds.get_engine()).get_table_names()
+        table_names: list[str] = get_engine_inspector(postgres_ds).get_table_names()
         print(f"postgres tables:\n{pf(table_names)}))")
 
         postgres_ds.add_table_asset(asset_name, table_name=table_name)
 
-    @pytest.mark.skip(reason="TODO: implement Databricks SQL integration service")
     @pytest.mark.databricks
     def test_databricks_sql(
         self,
         databricks_sql_ds: DatabricksSQLDatasource,
-        asset_name: str,
-        # table_factory: TableFactory,
+        asset_name: TableNameCase,
+        table_factory: TableFactory,
     ):
-        table_name = TABLE_NAME_MAPPING["databricks"].get(asset_name)
+        table_name = TABLE_NAME_MAPPING["databricks_sql"].get(asset_name)
         if not table_name:
             pytest.skip(f"no '{asset_name}' table_name for databricks")
+        # create table
+        table_factory(
+            engine=databricks_sql_ds.get_engine(),
+            table_names={table_name},
+            schema=PYTHON_VERSION,
+        )
 
-        table_names: list[str] = inspect(
-            databricks_sql_ds.get_engine()
-        ).get_table_names()
+        table_names: list[str] = get_engine_inspector(
+            databricks_sql_ds
+        ).get_table_names(schema=PYTHON_VERSION)
         print(f"databricks tables:\n{pf(table_names)}))")
 
-        databricks_sql_ds.add_table_asset(asset_name, table_name=table_name)
+        databricks_sql_ds.add_table_asset(
+            asset_name, table_name=table_name, schema_name=PYTHON_VERSION
+        )
 
     @pytest.mark.snowflake
     def test_snowflake(
         self,
         snowflake_ds: SnowflakeDatasource,
-        asset_name: str,
+        asset_name: TableNameCase,
         table_factory: TableFactory,
     ):
         table_name = TABLE_NAME_MAPPING["snowflake"].get(asset_name)
         if not table_name:
             pytest.skip(f"no '{asset_name}' table_name for databricks")
         # create table
+        schema = get_random_identifier_name()
         table_factory(
             engine=snowflake_ds.get_engine(),
             table_names={table_name},
-            schema=get_random_identifier_name(),
+            schema=schema,
         )
 
-        table_names: list[str] = inspect(snowflake_ds.get_engine()).get_table_names()
+        table_names: list[str] = get_engine_inspector(snowflake_ds).get_table_names(
+            schema=schema
+        )
```

Comment:
The `sqla.inspect()` command doesn't have return types defined and so I didn't notice that `get_table_names()` can take a `schema` param.